CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Contributing to the Phylum CLI
+
+Contributions are welcome and appreciated!
+
+Table of Contents:
+
+1. [Bug Reports](#bug-reports)
+2. [Feature Requests](#feature-requests)
+3. [Patches / Pull Requests](#patches--pull-requests)
+    1. [Testing](#testing)
+    2. [Documentation](#documentation)
+    3. [Style](#style)
+7. [Contact](#contact)
```

Comment:
This number is incorrect now. One tip I picked up from @kylewillmon is that, for numbered markdown lists like this, it is possible to simply use `1.` for _every_ entry. The rendered version will show the numbers correctly.